automation/vars/uni04delta.yaml

```diff
@@ -0,0 +1,54 @@
+---
+vas:
+  uni04delta:
+    stages:
+      - path: examples/dt/uni04delta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni04delta/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait osctlplane controlplane
+            --for condition=Ready
+            --timeout=600s
+        values:
+          - name: network-values
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni04delta/edpm-pre-ceph
```

Comment:
need to split as per https://github.com/openstack-k8s-operators/architecture/pull/197

Reply:
Agreed. We already have one DT landed without the split, and it'd be ideal to make this a habit before merging going forward.

Reply:
Dataplane split has been done